robot/CMakeLists.txt

```
@@ -29,23 +30,19 @@ if(${JSONCPP_LIBRARY_DIRS})
   include_directories("${JSONCPP_INCLUDE_DIRS}")
 endif(${JSONCPP_LIBRARY_DIRS})
 
-add_executable(robot ${SOURCES})
-target_link_libraries(robot
-  tcp
-  stdc++fs
-  wbt-translator
-  config
-  ${JSONCPP_LIBRARIES}
-  scheduling
-)
-
-install(TARGETS robot DESTINATION bin)
-
 add_library(librobot ${SOURCES})
 target_link_libraries(librobot
   tcp
   stdc++fs
+  config
   wbt-translator
-  ${JSONCPP_LIBRARIES}
   scheduling
+  ${JSONCPP_LIBRARIES}
```

Skal den rettes jf. vores nye `lib`-mappe?

```suggestion
  jsoncpp_lib
```